orchestrator/orchestrator/src/main_loop.rs

```diff
@@ -377,14 +362,9 @@ pub async fn eth_signer_main_loop(
                 e
             );
         }
-
-        // a bit of logic that tires to keep things running every LOOP_SPEED seconds exactly
-        // this is not required for any specific reason. In fact we expect and plan for
-        // the timing being off significantly
-        let elapsed = Instant::now() - loop_start;
-        if elapsed < ETH_SIGNER_LOOP_SPEED {
-            delay_for(ETH_SIGNER_LOOP_SPEED - elapsed).await;
-        }
+            },
+            tokio::time::sleep(std::time::Duration::from_secs(11))
```

Comment:
What's the justification for making this 11 seconds? Ethereum block time?

Reply:
The oracle and signer loops had their floor loop times defined here: https://github.com/PeggyJV/gravity-bridge/blob/2b96c5af1b9dadfa1f8596362a7653a8a8a3fa18/orchestrator/orchestrator/src/main_loop.rs#L40-L41

Last touched 9 months ago by Justin. I had thought they were related to block times somehow too, but then why would they be different? The commit message in the `git blame` sheds some light on his thoughts at the time:

https://github.com/PeggyJV/gravity-bridge/commit/65e4e6f6553b21e0c80ccd03f5b742be4e11bd0d

Reply:
I think what Zaki is referring to is me specifying 11 secs when I could've used the const specified here.
https://github.com/PeggyJV/gravity-bridge/blob/cd53921a0dcb388ab32c8061dea486a127d81554/orchestrator/orchestrator/src/main_loop.rs#L40

I already did that. If it's ok with you, I'll resolve the comment.